### ISSUE TITLE
Implementation of customModelPath for iOS

### ISSUE THREAD

**Issue by azharj96** [bharat-biradar/Google-Ml-Kit-plugin#77]
### Objective: 
Image labelling using custom model done in real-time. Should run on Android & iOS.

### Steps taken: 
When running the custom model on Android, everything works perfectly. The model path I use there is just `model.tflite` and the model is placed at `/android/app/src/main/assets/model.tflite`

When running on iOS, I use the path `model.tflite` (as on Android) with the model placed at `/ios/Runner/Resources/model.tflite`. But I get the following error:
```
*** Terminating app due to uncaught exception 'MLKInvalidLocalModel', 
reason: 'Local model path (model.tflite) is invalid.'
```

In the above example, I used the path `model.tflite` but I have also tried:
- `Resources/model.tflite`
- `Runner/Resources/model.tflite`
- `ios/Runner/Resources/model.tflite`

All of which result in an error like the above.

### What works:
I added the following snippet to [`getCustomLabelerOptions`](https://github.com/bharat-biradar/Google-Ml-Kit-plugin/blob/8788c341217dd4aff1e110c147646a495e0d4267/ios/Classes/vision/ImageLabeler.m#L79), ignoring the `path` param passed from flutter & the app starts working as expected!

```
    NSString *modelPath = [NSBundle.mainBundle pathForResource:@"model"
                                                        ofType:@"tflite"
                                                   inDirectory:@""];
    
    MLKLocalModel *localModel = [[MLKLocalModel alloc] initWithPath:modelPath];
```

Please let me know what I should be passing to the `customModelPath` param as part of the `CustomImageLabelerOptions` object to get the same result.

**Comment by bharat-biradar** [comment]
@azharj96 Can you clarify whether you are using, `v6.0.0` from pub.dev or directly using the code from master branch.

**Comment by azharj96** [comment]
@bharat-biradar I'm using the code directly from master branch as I need the fix done [here](https://github.com/bharat-biradar/Google-Ml-Kit-plugin/issues/55)

**Comment by bharat-biradar** [comment]
@azharj96 If you just want to use custom modelling and firebase remote model support pleade use `v6.0.0` from pub.dev. We recently updated the image labelling to support firebase remote model for android part and still needs to implemented for IOS. Readme also says currently image labelling for IOS is disabled. Let me know if this works.

**Comment by azharj96** [comment]
@bharat-biradar oh sorry I didn't see that! Thanks for the info.

I'm still getting the above error when using the package via the link you sent. What path should I be passing to `customModelPath`?

Also fyi, with the manual override change I mention in this issue + changing `custom` to `customLocal` on [this line](https://github.com/bharat-biradar/Google-Ml-Kit-plugin/blob/4c5d64f6e0d92fc98a73d3c48e6e40bf87e2d37d/ios/Classes/vision/ImageLabeler.m#L36), the image labelling worked for me.

**Comment by fbernaly** [comment]
We added this feature in these PRs for both iOS and Android:

* https://github.com/bharat-biradar/Google-Ml-Kit-plugin/pull/69
* https://github.com/bharat-biradar/Google-Ml-Kit-plugin/pull/99

We released both in 0.7.0.
